app/modules.go

```
@@ -0,0 +1,335 @@
+	"pkg.akt.dev/node/x/cert"
+	"pkg.akt.dev/node/x/deployment"
+	"pkg.akt.dev/node/x/escrow"
+	"pkg.akt.dev/node/x/market"
+	"pkg.akt.dev/node/x/provider"
+	"pkg.akt.dev/node/x/take"
+)
+
+func appModules(
+	app *AkashApp,
+	encodingConfig sdkutil.EncodingConfig,
+	skipGenesisInvariants bool,
```

golangci-lint (2.2.2)

[warning] unused-parameter: parameter 'skipGenesisInvariants' seems to be unused, consider removing or renaming it as _

(revive)